### ISSUE TITLE
Suggestion: Add a config

### ISSUE THREAD

**Issue by BLUEChester** [MrGenie151/Ender-Dragon-Fight-Remastered#3]
It would be great if we were able to customize the mod, as I personally would love to keep all the interesting attacks but lower the health back to 200. As it gets really repetitive shooting the dragon for that long when he has so much health.

Thanks

**Comment by BLUEChester** [comment]
Would definitely love to see it added but ofc its up to you and what you'd like to prioritize. I think you should also be able to turn attacks on/off and possibly edit the damage of the ender meteor the dragon shoots.

Like I said though, only add what you're interested in doing.

Thanks!

**Comment by BowBoy1** [comment]
Maybe have the ability to modify the time between attacks too.

**Comment by allanonmage** [comment]
I'm not following how summoning anvils is Dragon-like, so maybe make all or most of the changes configurable.  I'm putting things together for a vanilla style experience, and a better dragon fight would be cool, but the anvils don't make much sense for an improved vanilla/vanilla+ themed dragon fight.

**Comment by MrGenie151** [comment]
> I'm not following how summoning anvils is Dragon-like, so maybe make all or most of the changes configurable. I'm putting things together for a vanilla style experience, and a better dragon fight would be cool, but the anvils don't make much sense for an improved vanilla/vanilla+ themed dragon fight.

Yeah, i get that. Maybe ill replace that anvil one with something different idk

**Comment by Mr-iMikeyyy** [comment]
Hey guys, any update on the config? I'd like to be able to customize the dragon. My player's aren't fans of the falling anvils and lightning, I'd like to be able to turn those off.

**Comment by MrGenie151** [comment]
The anvils will likely be removed in the next update just because it isn't
a good attack

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by Nangt3c** [comment]
How can decrease the amount of angry endermen?

**Comment by Cath818** [comment]
Being able to remove the startup notification could be a good option for this!
